app/src/main/java/com/github/palFinderTeam/palfinder/meetups/fragments/MeetupFilterFragment.kt

```diff
@@ -49,6 +49,7 @@ class MeetupFilterFragment(val viewModel: MapListViewModel) : DialogFragment() {
             ShowParam.PAL_PARTICIPATING -> filterGroup.check(R.id.participate_button)
             ShowParam.PAL_CREATOR -> filterGroup.check(R.id.created_button)
             ShowParam.ONLY_JOINED -> filterGroup.check(R.id.joinedButton)
+            ShowParam.TRENDS -> filterGroup.check(R.id.trendButton)
```

Comment:
I see you managed to integrate it in the new design, but let me know if you need anything else!
Unless @TheblueMan003 wants to move it to the `order by` section in which case it will not appear in this fragment

Reply:
I think that's a design choice that we can discuss tomorrow and easily change !